uqcsbot/gaming.py

```diff
@@ -0,0 +1,241 @@
+from discord.ext import commands
+import discord
+import logging
+from uqcsbot.bot import UQCSBot
+from typing import List, Tuple, Optional
```

Comment:
Unneeded imports of logging, List, Tuple

Reply:
Fixed.